### PR TITLE
Added examples for Digital OPD flows

### PR DESCRIPTION
**Pull request by core-wg-admin**
Added examples for
- Discovery flows
- Order flows
- Fulfillment flows
for a Digital OPD fulfillment modality.

### CODE REVIEW

File: protocol-specifications/core/v0/api/core.yaml [hunk 1]
```diff
@@ -449,6 +864,71 @@ paths:
...
+                            end: T10:30Z
+                        customer:
+                          id: 61-8787-9274-4422
+                          cred: example@ndhm
+                      quote:
+                        price:
+                          currency: INR
+                          value: '120'
+                          breakup:
+                            - ./uhi-draft-0_9_3.consultation: '100'
```
Comment (AnirudhTiwari) [inline protocol-specifications/core/v0/api/core.yaml:922]: This tag should be dhp and not uhi

File: protocol-specifications/core/v0/api/core.yaml [hunk 2]
```diff
@@ -521,6 +1001,74 @@ paths:
...
+                            end: T10:30Z
+                        customer:
+                          ./nha.health_id: 61-8787-9274-4422
+                          ./nha.phr_address: example@ndhm
+                      quote:
+                        price:
+                          currency: INR
+                          value: '120'
+                          breakup:
+                            - ./uhi-draft-0_9_3.consultation: '100'
```
Comment (AnirudhTiwari) [inline protocol-specifications/core/v0/api/core.yaml:1063]: Same as [1]

File: protocol-specifications/core/v0/api/core.yaml [hunk 3]
```diff
@@ -896,7 +1783,7 @@ components:
               $ref: '#/components/schemas/Person'
             contact:
               $ref: '#/components/schemas/Contact'
-        agent:
+        person:
```
[the remark this answers is not in the document]
Comment (core-wg-admin) [reply protocol-specifications/core/v0/api/core.yaml:1786]: Not yet. We may have to keep it as a non-standard attribute - `./dhp-0_7_1.person` for now as the `Agent` schema in beckn has a `Person` schema embedded in it. Will make that change

File: protocol-specifications/core/v0/api/core.yaml (hunk 1 above)
Comment (core-wg-admin) [reply protocol-specifications/core/v0/api/core.yaml:922]: Noted. Will make that change.

File: protocol-specifications/core/v0/api/core.yaml (hunk 2 above)
Comment (core-wg-admin) [reply protocol-specifications/core/v0/api/core.yaml:1063]: Noted. Will make that change.

File: protocol-specifications/core/v0/api/core.yaml (hunk 1 above)
Comment (core-wg-admin) [reply protocol-specifications/core/v0/api/core.yaml]: This is done

File: protocol-specifications/core/v0/api/core.yaml (hunk 2 above)
Comment (core-wg-admin) [reply protocol-specifications/core/v0/api/core.yaml]: This is done

File: protocol-specifications/core/v0/api/core.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (core-wg-admin) [reply protocol-specifications/core/v0/api/core.yaml:1786]: Ignore the above comment. Have added this in the `core-0.9.3-release` branch as well.

**Pull request merged**